aiida/engine/transports.py

```diff
@@ -41,7 +40,6 @@ class TransportQueue:
     up to that point.  This way opening of transports (a costly operation) can
     be minimised.
     """
-    AuthInfoEntry = namedtuple('AuthInfoEntry', ['authinfo', 'transport', 'callbacks', 'callback_handle'])
```

Comment:
this is not used anywhere in the code base. Is it necessary?

Reply:
looks like this was introduced by @muhrin in https://github.com/aiidateam/aiida-core/commit/776d3a359c025f66c65b0d25be8e6cdc97e88239 (Jul 2017)

If it hasn't been used in 3.5 years, it would seem that this can be removed.
I'd like to wait for an OK from either @sphuber or @muhrin on this though

Reply:
@sphuber / @muhrin can you drop a quick approval, then I can merge 😬

Reply:
haha, please drop away.